src/main/resources/assets/alexsmobs/lang/zh_hk.json

```diff
@@ -0,0 +1,648 @@
+  "alexsmobs.sound.subtitle.snow_leopard_idle": "雪豹嗷嗚",
+  "alexsmobs.sound.subtitle.snow_leopard_hurt": "雪豹怒吼",
+  "alexsmobs.sound.subtitle.spectre_idle": "靈魅嚎叫",
+  "alexsmobs.sound.subtitle.spectre_hurt": "靈魅尖叫",
+  "alexsmobs.sound.subtitle.crow_idle": "烏鴉哇哇叫",
+  "alexsmobs.sound.subtitle.crow_hurt": "烏鴉尖叫",
+  "alexsmobs.sound.subtitle.alligator_snapping_turtle_idle": "大鱷龜嘶嘶聲",
+  "alexsmobs.sound.subtitle.alligator_snapping_turtle_hurt": "大鱷龜齜牙咧嘴",
+  "alexsmobs.sound.subtitle.mungus_idle": "蘑菇君低鳴",
+  "alexsmobs.sound.subtitle.mungus_hurt": "蘑菇君喘息",
+  "alexsmobs.sound.subtitle.mungus_laser_loop": "蘑菇君射擊",
+  "alexsmobs.sound.subtitle.mungus_laser_end": "蘑菇君射擊結束",
+  "alexsmobs.sound.subtitle.mantis_shrimp_hurt": "螳螂蝦裂開",
+  "alexsmobs.sound.subtitle.mantis_shrimp_snap": "螳螂蝦咬住",
+  "alexsmobs.sound.subtitle.guster_idle": "沙風怪呼呼吹",
+  "alexsmobs.sound.subtitle.guster_hurt": "沙風怪嗖嗖聲",
+  "alexsmobs.sound.subtitle.warped_mosco_idle": "迷離蚊鬼碰撞聲",
+  "alexsmobs.sound.subtitle.warped_mosco_hurt": "迷離蚊鬼嗡嗡聲",
+  "alexsmobs.sound.subtitle.straddler_idle": "跨座獸咆哮",
+  "alexsmobs.sound.subtitle.straddler_hurt": "跨座獸尖叫",
+  "alexsmobs.sound.subtitle.emu_idle": "鴯鶓跳動",
+  "alexsmobs.sound.subtitle.emu_hurt": "鴯鶓尖叫",
+  "alexsmobs.sound.subtitle.platypus_idle": "鴨嘴獸吱吱聲",
+  "alexsmobs.sound.subtitle.platypus_hurt": "鴨嘴獸低吼",
+  "alexsmobs.sound.subtitle.dropbear_idle": "懸熊喘息",
+  "alexsmobs.sound.subtitle.dropbear_hurt": "懸熊尖叫",
+  "alexsmobs.sound.subtitle.tasmanian_devil_idle": "袋獾低吼",
+  "alexsmobs.sound.subtitle.tasmanian_devil_hurt": "袋獾嘶嘶聲",
+  "alexsmobs.sound.subtitle.tasmanian_devil_roar": "袋獾尖叫",
+  "alexsmobs.sound.subtitle.kangaroo_idle": "袋鼠咳聲",
+  "alexsmobs.sound.subtitle.kangaroo_hurt": "袋鼠哭泣",
+  "alexsmobs.sound.subtitle.cachalot_whale_idle": "抹香鯨哢嚓聲",
+  "alexsmobs.sound.subtitle.cachalot_whale_hurt": "抹香鯨哭泣",
+  "alexsmobs.sound.subtitle.cachalot_whale_click": "抹香鯨回聲",
+  "alexsmobs.sound.subtitle.leafcutter_ant_hurt": "切葉蟻吧唧聲",
+  "alexsmobs.sound.subtitle.leafcutter_ant_queen_hurt": "切葉蟻後哭泣",
+  "alexsmobs.sound.subtitle.enderiophage_hurt": "終界噬菌體尖叫聲",
+  "alexsmobs.sound.subtitle.enderiophage_walk": "終界噬菌體漫步",
+  "alexsmobs.sound.subtitle.enderiophage_squish": "終界噬菌體發射",
+   "alexsmobs.sound.subtitle.bald_eagle_idle": "白頭海鵰嚎叫",
+  "alexsmobs.sound.subtitle.bald_eagle_hurt": "白頭海鵰尖叫",
+  "alexsmobs.sound.subtitle.tiger_idle": "老虎咆哮",
+  "alexsmobs.sound.subtitle.tiger_hurt": "老虎哼哼",
+  "alexsmobs.sound.subtitle.tiger_angry": "老虎怒吼",
+  "alexsmobs.sound.subtitle.tarantula_hawk_wing": "沙漠蛛蜂嗡嗡聲",
+  "alexsmobs.sound.subtitle.tarantula_hawk_hurt": "沙漠蛛蜂嘶嘶聲",
+  "alexsmobs.sound.subtitle.music_disc.thime": "Thime",
+  "alexsmobs.sound.subtitle.music_disc.wormboss": "Wormboss",
+  "alexsmobs.sound.subtitle.void_worm_idle": "虛空蠕蟲低鳴",
+  "alexsmobs.sound.subtitle.void_worm_hurt": "虛空蠕蟲嘶嘶聲",
+  "alexsmobs.sound.subtitle.void_portal_open": "虛空傳送門打開",
+  "alexsmobs.sound.subtitle.void_portal_closes": "虛空傳送門關閉",

```

Comment:
The correct string is "alexsmobs.sound.subtitle.void_portal_close" (without the extra s at the end)
This is fixed for other translations in #568